app/views/applications/show.html.erb

```diff
@@ -3,5 +3,19 @@
 <p>Name: <%= @app.name %></p>
 <p>Address: <%= @app.full_address %></p>
 <p>Description: <%= @app.description %></p>
-<p>Pets: <%= @app.pet_names %></p>
-<p>Application Status: <%= @app.status %></p>
\ No newline at end of file
+<p>Pets: <%= @app.pet_names.join(" | ") %></p>
+<p>Application Status: <%= @app.status %></p>
+
+<% if @app.status == "In Progress" %>
+  <hr>
+  <h3>Add a Pet</h3>
+  <%= form_with url: "/applications/#{@app.id}", method: :get, turbo: false do |form| %>
+    <%= form.label :q, "Name:" %>
+    <%= form.search_field :q %>
```

Comment:
Curious as to why use :q as the search param? Is this from a pm background as we can explicitly define it as :search

Reply:
:q is the convention for query, Juliet showed the example in class as such, but if it helps you call it :search I won't die on that hill

Reply:
Nice catch. Lets go with q since it's already implemented successfully. It's always preferable for convention